### DISCUSSION TITLE
EXE file generated by jpackage have any WIX related files ?

### DISCUSSION THREAD

**Discussion by subramaniSCB** [wixtoolset/issues#7838] [Questions]
When we create an installer file using jpackage command in WIX binary, .exe file which is generated will have any WIX tool related files in the package ?

we are using following command to generate the executable file. 
jpackage --name Test --input "..\files" --main-jar testoutput.jar

**Comment by chrpai** [comment]
I've never used jpackage so without seeing the wxs it generates or the resulting packge it's hard to say.  It won't install any wix files thats for sure.  But if it's using extensions (for example wix_ui)  or creating a boostrapper there will most certainly be some wix artifacts contained in the package.
